### ISSUE TITLE
"Update and Shut Down" script

### ISSUE THREAD

**Issue by phillipzink** [mkellyxp/nixbook#9]
Add an "Update and Shut Down"  script similar to "Update and Reboot", so that we can shut down at the "end of the week" and come back to an updated computer at the "beginning of the week" without having to wait on the computer to do maintenance tasks.

**Comment by mkellyxp** [comment]
Great idea.  I also just realized an issue I tagged you in Masoton as well.

These manual update scripts just manually run nixos-rebuild.  However, when we switch the channel from the auto update, it doesn't actually switch the channel that nixos-rebuild will use.  

I might have to change this so it will manually run the sytemd-timer job for the autoupdate instead of nixos-rebuild.  Will have to do some playing around with that!

**Comment by phillipzink** [comment]
That was issue #7

I mostly copied update.sh and then called systemctl poweroff at the end instead of reboot. I'd put them in one script, but that would break people's Update and Shutdown until they did a Powerwash or full install.

[https://github.com/phillipzink/nixbook/blob/nixos-and-flatpak-update-fixes/update_shutdown.sh](https://github.com/phillipzink/nixbook/blob/nixos-and-flatpak-update-fixes/update_shutdown.sh)

**Comment by mkellyxp** [comment]
This has been tested and merged into main!  Thank you!
